CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/pusher/push-notifications-swift/compare/1.2.0...HEAD)
+## [Unreleased](https://github.com/pusher/push-notifications-swift/compare/1.3.0...HEAD)
+
+## [1.3.0](https://github.com/pusher/push-notifications-swift/compare/1.2.0...1.3.0) - 2019-02-20
+
+## Added
```

Comment:
I think this is more of a fix, so 1.2.1

Reply:
I don't see this as a bug fix. We've added a new functionality in a backwards compatible manner that's why it's `1.3.0`.